code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -1017,3 +1020,6 @@
 
 /mob/living/silicon/ai/zMove(dir, feedback = FALSE)
 	. = eyeobj.zMove(dir, feedback)
+
+/mob/living/silicon/ai/is_malf()
+	return is_special_character(src)
```

Comment:
Further up, you use this as if it's comparable to 
```
if(owner?.mind?.special_role)
		return
	if (istype(owner, /mob/living/silicon/ai))
		var/mob/living/silicon/ai/A=owner
		if(A?.deployed_shell?.mind?.special_role)
			return
```
Is there a check I'm missing?

Reply:
I don't understand.